src/build.rs

```diff
@@ -39,12 +51,18 @@ fn main() {
     let ext = "so";
     let source = format!("{prefix}{name}.{ext}");
     let target = format!("{prefix}{name}{suffix}.{ext}");
+    symlink(&path, &source, &target);
```

Comment:
this symlink fails when running `cargo build` on a static library, with `No such file or directory`. Should we do the symlink only if we create the dynamic library?

Reply:
Fixed by adding `std::fs::create_dir_all(path).unwrap();` in `symlink` function!